ubcpi/static/html/ubcpi.html

```diff
@@ -4,12 +4,12 @@
 
         <div class="question-other-explain" data-ng-if="rc.status() == rc.ALL_STATUS.NEW || rc.status() == rc.ALL_STATUS.ANSWERED">
 
-            <h2 id="pi-question-h" class="question-text">{{display_name}}</h2>
+            <h3 id="pi-question-h" class="question-text" style="display:inline;text-transform:uppercase">{{display_name}}</h3>
```

Comment:
all uppercase is generally frowned upon.  It has accessibility and usability implications.  We don't use it anywhere else in the platform.   Research has shown that all caps is harder for the general population to read and parse.  This difficulty is magnified for users with certain cognitive disorders.  It would be great if you could stick with the style consistent with other H3s in course content.